e2e_tests/account_test.py

```diff
@@ -88,3 +88,32 @@ def test_update_account():
     response = client.accounts.update(request)
     assert response.data.type == "depositAccount"
 
+def test_get_deposit_products():
```

Comment:
if nothing will be returned the test will also pass.. we need to validate that something returned